lua/wire/wireshared.lua

```diff
@@ -1016,7 +1016,8 @@ end)
 -- Massive numbers applied through E2, which are allowed by the code right above, may change the direction of the applyForce
 function WireLib.clampForce( v )
 	-- v, whether it be a vector or angle, can be indexed like a table, so we just check each value, modify it, then return v
-	-- check if each value is nan then make it 0 if it is, else clamp it
+	-- Lua Ternary: Standard    http://lua-users.org/wiki/TernaryOperator
```

Comment:
You don't need to comment this, it's a bit redundant considering it's somewhat a standard language feature that's used quite often.